site/src/request_handlers/github.rs

```diff
@@ -138,13 +137,30 @@ async fn handle_rust_timer(
         return Ok(github::Response);
     }
 
-    for captures in build_captures(&comment.body).map(|(_, captures)| captures) {
-        let include = captures.get(2).map(|v| v.as_str());
-        let exclude = captures.get(3).map(|v| v.as_str());
-        let runs = captures.get(4).and_then(|v| v.as_str().parse::<i32>().ok());
-        {
-            let conn = ctxt.conn().await;
-            conn.queue_pr(issue.number, include, exclude, runs).await;
+    let build_cmds: Vec<_> = parse_build_commands(&comment.body).collect();
```

Comment:
probably don't need to collect here

(nor really need the vec in the first place since it's iterated twice anyways, but it's fine ^^)

Reply:
Thanks, the code was a bit different before, but then I removed the second usage of the Vec.